.github/workflows/maven-publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish the Java Package to Maven Central Repository
+
+on:
+  push:
+    tags:
+      - java-[0-9]+.[0-9]+.[0-9]+
```

Comment:
If we manage versions for both Java and Node, should we separate the repo?
Or we should have a single version for the admin and release both Java and Node clients?
Managing different versions (with the same version number) is weird to me.
(node-1.2.0 and java-1.2.0 could be created from the different code?)

Reply:
@feeblefakie 

Thank you for your opinion.

I separated the versions in case some issue happens in, for example, the Node client library but not in the Java client library then we can bump the bugfix version for the Node client library release only.

But, I rethink it again. Always releasing them with the same versions is easier for maintenance and can avoid confusion. Even if the bugfix doesn't change anything on, for example, the Java client library, bumping the bugfix doesn't really violate the versioning convention.

I will create another PR to unify the CI.

Reply:
OK, sounds good. Thank you!